### ISSUE TITLE
Logo proposal

### ISSUE THREAD

**Issue by jbeguna04** [NilsFo/LockScreenNotes#1]
Good day @NilsFo  , I am a graphics designer and I would like to help you improve your logo/icon for your project for your future use. I would be glad to help open source projects like this. Thank you in advance and have a great day ahead :)

Yours truly,
jbeguna04

**Comment by NilsFo** [comment]
Dear @jbeguna04 

Thanks for reaching out to me. As you have already noted this is project is open source and non-profit. Thus, I doubt I'll be able to offer you any monetary compensation.
Otherwise, I'd be happy to hearing from you about the inquiry.

Regards,
Nils

**Comment by jbeguna04** [comment]
No, this proposal is totally free and not asking any monetary compensation, I am honored to contribute on your project and offer help for your future needs.
 
Best regards,
jbeguna04

**Comment by jbeguna04** [comment]
![image](https://user-images.githubusercontent.com/35353768/47483526-0f538700-d86c-11e8-8ecc-6a16141ccf58.png)

Here is my proposal design for the project, What do you think sir?

**Comment by NilsFo** [comment]
Sorry for the late reply.
Those look really good!!
But I am currently very occupied with my job, so I'll get back to you on those at a later time. I hope that's okay!

**Comment by jbeguna04** [comment]
Sure , If you want sir I can add it on your repository/files by making a PR but is still your choice. Thank you for your appreciation
